user_guide_src/source/incoming/controllers/017.php

```diff
@@ -1,15 +1,18 @@
 <?php
 
-public function updateUser(int $userID)
+class userController extends BaseController
```

Comment:
```suggestion
class UserController extends BaseController
```

Reply:
According to this:
https://codeigniter4.github.io/userguide/incoming/controllers.html#let-s-try-it-hello-world
Only the first letter should be capital. Is that correct?
I always though we use CamelCase.

Reply:
It is the convention from CodeIgniter3, and it is true for only auto-routing.
Normally a URI is all lower case, and CodeIgniter converts the first letter to capital.
Therefore `/helloworld` is mapped to `Helloworld` class.

If you use defined routes, CamelCase is no problem. 
Because you define the URI in `Routes.php`.